pages/index/index.js

```diff
@@ -3,6 +3,7 @@ import glamorous from 'glamorous'
 import {LiveProvider, LiveEditor, LiveError} from 'react-live'
 import {LiveContextTypes} from 'react-live/lib/components/Live/LiveProvider'
 import Head from 'next/head'
+import {AlgoliaLink, AlgoliaScript} from '../../components/algolia-config'
```

Comment:
Ahh, just tried that and it looks like the next Head component doesn't like multiple Heads.

One option could be to change the shape of how Layout is used to instead have a render method which passes PageHead as a prop

```
// Layout.js
   const PageHead = () => (
        <Head>
            <AlgoliaLink />
            <AlgoliaScript />
            {children}
        </Head>
    )
   ...
    <Div overflow="auto" width="100%">
        {renderChildren({PageHead})}
    ...

// Page.js
<Layout
    render={({PageHead}) => {
        ...
            <PageHead>
                ...
```

happy for other suggestions though